### DISCUSSION TITLE
Not connecting any more over mobile data (new provider) - ipv6 related?

### DISCUSSION THREAD

**Discussion by streiman** [owntracks/android#1113] [Q&A] [answered]
I changed my mobile data provider (Germany, from Vodafone to O2) and I changed my Android phone as well.

Since then the owntracks app can only reach my owntrack server hosted on my intel nuc (domain owntracks.<mydomain>.me) when I am using my WLAN. Outside the house, connected by O2 5G I get an error:

Failed to connect to owntracks.<mydomain>.me/
2001:16b8:<4digits>:<4digits>:<3digits>:<4digits>:<4digits>:<4digits>:443

Before using vodafone mobile data everything worked well for many, many months.

I guess it is ipv6 configuration related, but I really don't know where to look and/or check or what to change.

Could anybody point me in the right direction?

**Comment by jpmens** [comment, marked as answer]
It appears as thought the domain name you are using has an IPv6 address in the DNS (AAAA record) which applications always use first; they will typically not fall back to  using IPv4 (A record) if a AAAA is configured.

So, if you know you have no IPv6 on the target system and/or if you know O2 doesn't provide support for IPv6, the easiest (likely only) solution would be to remove the AAAA record from the DNS.

**Reply by streiman** [reply]
I use synology.me to redirect from my domain to my homeserver, added as cname to my domain's DNS. It seems I am not able to remove the IPv6 record from the DNS.

Do you know any other way or point me in a direction to configure owntracks on my target system for IPv6?

**Reply by growse** [reply]
Does your home network use IPv6?

It's not 100% clear to me what the DNS setup is here. You have the domain that you configured in OT app (e.g. owntracks.example.me), which is the CNAME'd to another domain. Who controls that target domain?

It seems that Synology.me is a dynamic-dns service, so it's weird that they're populating a AAAA record if you don't have IPv6 on your NUC.

**Reply by streiman** [reply]
If own owntracks.example.me. In the DNS record I added <myname>.synology.me as CNAME. It is a dynamic-dns-service, provided by Synology. I really don't know which records they populate and it seems that there is nothing to change as options.

On my NUC I am not sure what I have activitated, there are several services running and I had no trouble with any of them. If I open owntracks.example.me in Chrome on my Android phone everything works fine, but not the updates of the owntracks Android app.

**Reply by NeilJared** [reply]
Would someone be so glad to share step-by-step guide on mounting all this on a Synology DSM 7? Many of us have little to no skills, but would like to dive on NAS usage and ejoy OwnTracks too. Please, share this... I can only help translating into my mother languaje (SPA), just drop me a message.

**Comment by jpmens** [comment]
Would you care to send the real domain name to us at ***@***.*** ? I can query the name and determine whether what I mentioned could be an issue. Alternatively, if you have a dig utility you can do so yourself:

dig owntracks.example.me AAAA
dig myname.synology.me AAAA

**Reply by streiman** [reply]
I solved the problem by disabling IPv6 on my Symology NAS. Obviously this changes the symology.me DNS settings - and now everything works as before.

Thanks for all the support.

**Reply by jpmens** [reply]
thanks for the report; glad you got it working.

**Comment by kri164** [comment]
IPv6 is working in my case.
Server has A/AAAA  adress in DNS, dualstack connectivity and mobile connection from O2 has IPv6 too.
